usr/share/rear/conf/default.conf

```diff
@@ -2143,8 +2143,26 @@ GALAXY10_BACKUPSET=
 GALAXY10_Q_ARGUMENTFILE=
 ####
 
-####
-# BACKUP=TSM
+# BACKUP=GALAXY11
+##
+# Note: This is for CommVault 11
+#
+COPY_AS_IS_GALAXY11=( /opt/commvault/ /etc/CommVaultRegistry )
+COPY_AS_IS_EXCLUDE_GALAXY11=( "/opt/commvault/iDataAgent/jobResults/*" "/opt/commvault/Updates/*" "/opt/commvault/Base64/Temp/*" )
+# Use this default backup set. If left empty we will prompt the user at restore time for
+# the appropriate backup set to use. Could be set to a global organisation-wide default
+# in site.conf ...
+GALAXY11_BACKUPSET=
+# Use this argument file for all Commvault commands. You can place some general stuff here
+# and also use this file to store the logon credentials to Commvault. This file will be
```

Comment:
FYI in general regarding "credentials":
You may have a look at the UserInput function description
about "confidential user input mode" what it describes
how to call commands that deal with confidential data
in an appropriate (temporary) environment
e.g. with STDERR redirected to /dev/null
in particular to avoid that confidential data leaks out
into ReaR's log file in debugscript ('set -x') mode, see
https://github.com/rear/rear/blob/rear-2.7/usr/share/rear/lib/_input-output-functions.sh#L973

Reply:
`qlogin` asks for interactive credentials, not ReaR, so nothing for us to worry about